README.md

```diff
@@ -1377,3 +1377,10 @@ I am sure it will be an awesome journey!
 # Hello there!
 1st time working via command line, [Vasiliy](https://github.com/Vas-P) =)
 here's a potato ![Potato](https://blog.printsome.com/wp-content/uploads/potato-printing.jpg)
+
+---
+# Hello, Kottans! 
+
+I`m *[Hanna](https://github.com/HannaSyn)*, and I am really excited to join community! :smiling_face_with_three_hearts:
+[kitten](https://slm-assets.secondlife.com/assets/16934237/view_large/animals_05.jpg?1492508022) 
```

Comment:
Hi Hanna!
Awesome! 👍 
But if you want to embed an image in markdown you can try to use such template
`![image description](image-link.jpeg)`

Reply:
More info [here](https://www.markdownguide.org/basic-syntax/#images)

Reply:
> Hi Hanna!
> Awesome! 👍
> But if you want to embed an image in markdown you can try to use such template
> `![image description](image-link.jpeg)`

I mean `!` symbol 😄

Reply:
Thank you! Got it :sweat_smile: